### ISSUE TITLE
Open uploader's profile with long click

### ISSUE THREAD

**Issue by avently** [TeamNewPipe/NewPipe#814]
- [+] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [+] I checked if the issue/feature exists in the latest version.
I wanted to add onlongclick listener to related items below a video fragment and then open an uploader's channel (when action will occur) . But extractor's StreamInfoItem.java doesn't contain one important field - uploader's channel URL. So i can't open it without the url. Can you include this field into StreamInfoItem? Of course would be cool to have this feature (open channel with long click on the related video) out-of-the-box but it's ok if you are not interested in it. I can easily add longClicked method into your OnInfoItemSelectedListener interface and this will add more possibilities like showing menu with multiple items (if you will need it in the feature).

**Comment by wb9688** [comment]
No, you could convert it to an URL with ChannelUrlIdHandler#getUrl iirc.

**Comment by avently** [comment]
@wb9688 I don't see a working solution in your answer.
StreamInfoItem contains url to YouTube video like https://www.youtube.com/watch?v=Wj8kLUoUZ6A
Channelurl should contain /user or /channel path. I can't convert YouTube video url to channel url with YouTubechannelurlidhandler.
Am I wrong?

**Comment by wb9688** [comment]
@avently: Oh, sorry, I misunderstood. I don't think it'd be that hard to
add, could you maybe add it yourself?

@theScrabi: Could you open this again?

**Comment by theScrabi** [comment]
To be honest, I am not a huge fan of this. Because again this would also open a second pathway to the channel, and also requires change in the extractor. I'm trying to not add to much extra functionality to the extractor until we finally have multi service support  in order not to brake multi service.

**Comment by avently** [comment]
@wb9688 thank you for your pull request.;  @theScrabi you will understand how helpful to use this feature once you'll try it.

**Comment by avently** [comment]
@theScrabi can you review the pull request @wb9688 made? After that I will add a code related to longpress action to my pull request with huge code rewriting (player will work in videoplayerfragment without using an activity)
